### PR TITLE
Separating Scala Actors from the Scala Library.

### PR DESCRIPTION
**Pull request by vjovanov**
Scala actors are now in scala-actors.jar. Changes that were done are:
- Fixed partest to include actors library for various test usages
- Created the entry for the new jar in build.xml
- Added maven entries for scala actors

Review by: @jsuereth

### CODE REVIEW

**Comment by jsuereth** [comment]
So, +10 on modularizing out actors.   Quick question:  Are you creating a `scala-actors-docs.jar` with the scaladoc?  I didn't catch it in the diff.

**Comment by vjovanov** [comment]
Great catch, I missed it and it is fixed now. Currently we are copying the scala-library-doc.jar to be the same as scala-swing-doc.jar so I made the same hack for actors. Not sure should if this should be changed.

**Comment by jsuereth** [comment]
It should, the issue is build time.  Scaladoc adds a lot, and this is "good
enough" for now....
On Mar 19, 2012 12:18 PM, "Vojin Jovanovic" <
reply@reply.github.com>
wrote:

File: src/build/maven/scala-actors-pom.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+   <licenses>
+      <license>
+         <name>BSD-like</name>
+         <url>http://www.scala-lang.org/downloads/license.html
+         </url>
+         <distribution>repo</distribution>
+      </license>
+   </licenses>
+   <scm>
+      <connection>scm:svn:http://lampsvn.epfl.ch/svn-repos/scala/scala/trunk</connection>
```
Comment (soc) [inline src/build/maven/scala-actors-pom.xml:26]: Should probably point to https://github.com/scala/scala.

File: src/build/maven/scala-actors-pom.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      <url>https://lampsvn.epfl.ch/trac/scala/browser/scala/trunk</url>
+   </scm>
+   <issueManagement>
+      <system>trac</system>
+      <url>http://lampsvn.epfl.ch/trac/scala
```
Comment (soc) [inline src/build/maven/scala-actors-pom.xml:31]: Should probably point to https://issues.scala-lang.org.

Comment (jsuereth) [inline src/build/maven/scala-actors-pom.xml:31]: Good call.  Thanks.

Comment (vjovanov) [inline src/build/maven/scala-actors-pom.xml:31]: I know. I did not change this as it needs to be changed in multiple places. I will make a separate pull request for that.

File: src/build/maven/scala-actors-pom.xml (hunk 1 above)
Comment (vjovanov) [inline src/build/maven/scala-actors-pom.xml:26]: I will fix it in a separate pull request. There are multiple places where this fix needs to be applied. I was keeping it consistent and separating concerns.

**Pull request merged**